fcrepo-clients/src/main/java/edu/unc/lib/dl/fcrepo4/TransactionalFcrepoClient.java

```diff
@@ -93,6 +95,11 @@ public static TransactionalFcrepoClientBuilder client(String baseUri) {
     @Override
     public FcrepoResponse executeRequest(URI uri, HttpRequestBase request)
```

Comment:
final URI uri ?

Reply:
i just thought it would make it explicit that the param itself isn't being changed since you immediately re-assign it to a local var. not a big deal, though :-)